### DISCUSSION TITLE
How is CppSharp compared to SharpGenTools?

### DISCUSSION THREAD

**Discussion by ghost** [mono/CppSharp#1784] [Q&A] [answered]
https://github.com/SharpGenTools/SharpGenTools

It has been used on various projects, the most notable I know is https://github.com/amerkoleci/Vortice.Windows.

**Comment by tritao** [comment, marked as answer]
There is a previous comparison here which was written years ago: https://github.com/mono/CppSharp/blob/main/docs/DevManual.md#similar-tools

Not really sure how much things have changed but might be helpful.
